### ISSUE TITLE
As a  Christmas tree permit customer, I want to get a permit on the last day of the season so that I can get a permit at my convenience.

### ISSUE THREAD

**Issue by ASprinkle** [USDAForestService/fs-open-forest-platform#1131]
## Notes

On 12/31 the Christmas tree site showed the Oka-Wen, Umatilla, Gifford Pinchot, and Flathead as closed for the season, no ability to "buy permit" on the last day of the season..  These forest's end of season date is 12/31.  The same instance occurred for the Arapaho and Roosevelt, and the Willamette on January 6, their end of season date.
![image.png](https://images.zenhubusercontent.com/5a00e34b8a75884b90864b6f/7a68c09c-d00e-4a5f-b679-3c28af1d4c9d)

Forests confirmed that the expectation is for people to be able to get permits "through" the end of the season date, so the system should be issuing permits through midnight on the end of the season date for the respective forest.  Permits would then not be available for purchase on the day after the end of the season.

## Acceptance Criteria
- [ ] Someone can buy a christmas tree permit on the last day of the season.

## Tasks
- [ ] Fix so that the season date end date for each forest is through 2359 (11:59 PM)
- [ ]  Confirm that the season start date for each forest begins at 0001 (12:01AM)
- [ ] PO approved

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Usability tested

**Comment by ASprinkle** [comment]
In response to task: "Confirm that the season start date for each forest begins at 0001 (12:01AM)" the answer is Yes, within their own time zones.

**Comment by briandavidson** [comment]
@ASprinkle I've got the fix for this issue, but it's revealed another smaller issue that can be seen here: 
![image](https://user-images.githubusercontent.com/5187168/73775055-29f4fb00-4753-11ea-980c-acff6b069b62.png)

If it's alright with the team I'd like to address this in a separate follow up issue since while it is in the same area, this is a different issue altogether.

**Comment by ASprinkle** [comment]
@briandavidson agree. separate issue/separate tasks.  thanks!

**Comment by ASprinkle** [comment]
Tested on all forests, and is working well. All forests offered the buy permit button on the end of season date (2/12 in these test cases).
![image](https://user-images.githubusercontent.com/33435621/74360429-e8c7a100-4d79-11ea-9950-d38004acfb66.png)

**Comment by mtlaney** [comment]
blocked by #1196
